guides/join_space_mutation.md

```diff
@@ -0,0 +1,50 @@
+---
+copyright: 'Copyright IBM Corp. 2017'
+link: 'join-space-mutation'
+is: 'experimental'
+---
+
+# Join Space
+
+## Concepts
+
+The _joinSpace_ mutation allows you to join a workspace, where applicable, with the space ID. Currently this only applicable to spaces you are currently invited to.
```

Comment:
what does the qualifier "where applicable" mean? 

The _joinSpace_ mutation allows the user to join a space. Currently this is only applicable to spaces the user is invited to already.

Reply:
It means that joining spaces is only applicable for now when the user has an invite.

Reply:
could you use the suggested sentences I've provided?